### PR TITLE
[HOPSWORKS-718] Feature Store in HopsML

### PR DESCRIPTION
**Pull request by Limmen**
- Add tf-spark-connector for converting between spark dataframes and TFRecords

### CODE REVIEW

**Comment by Limmen** [comment]
Tests: https://github.com/logicalclocks/hops-testing/pull/248

File: attributes/default.rb [hunk 1]
```diff
@@ -377,6 +377,12 @@
 default['hops']['hopsutil_version']                    = node['install']['version']
 default['hops']['hopsutil']['url']                     = "#{node['download_url']}/hopsutil/#{node['hops']['hopsutil_version']}/hops-util-#{node['hops']['hopsutil_version']}.jar"
 
+#
+# Tensorflow-Spark Connector (Featurestore dependency)
+#
+default['hops']['tf_spark_connector_version']                            = "2.11-1.12.0"
+default['hops']['tf_spark_connector']['url']                     = "#{node['download_url']}/spark-tensorflow-connector_#{node['hops']['tf_spark_connector_version']}.jar"
+
```
Comment (SirOibaf) [inline attributes/default.rb]: Why didn't you put these in the `spark-chef` cookbook?

Comment (Limmen) [reply attributes/default.rb]: @SirOibaf No particular reason, just followed the same practice as hops-util.jar and hops-spark-examples.jar, do you want me to move it?

Comment (tkakantousis) [reply attributes/default.rb]: Prehistorical reason actually. I agree

Comment (Limmen) [reply attributes/default.rb]: This is fixed now, I will close this PR, please add comments to spark-chef PR if you have more feedback.

**Pull request closed without merging**